README.md

```diff
@@ -1,6 +1,17 @@
 # EdgeX Secrets Module
+
 [![Build Status](https://jenkins.edgexfoundry.org/view/EdgeX%20Foundry%20Project/job/edgexfoundry/job/go-mod-secrets/job/main/badge/icon)](https://jenkins.edgexfoundry.org/view/EdgeX%20Foundry%20Project/job/edgexfoundry/job/go-mod-secrets/job/main/) [![Code Coverage](https://codecov.io/gh/edgexfoundry/go-mod-secrets/branch/main/graph/badge.svg?token=KrqJoby1fK)](https://codecov.io/gh/edgexfoundry/go-mod-secrets) [![Go Report Card](https://goreportcard.com/badge/github.com/edgexfoundry/go-mod-secrets)](https://goreportcard.com/report/github.com/edgexfoundry/go-mod-secrets) [![GitHub Latest Dev Tag)](https://img.shields.io/github/v/tag/edgexfoundry/go-mod-secrets?include_prereleases&sort=semver&label=latest-dev)](https://github.com/edgexfoundry/go-mod-secrets/tags) ![GitHub Latest Stable Tag)](https://img.shields.io/github/v/tag/edgexfoundry/go-mod-secrets?sort=semver&label=latest-stable) [![GitHub License](https://img.shields.io/github/license/edgexfoundry/go-mod-secrets)](https://choosealicense.com/licenses/apache-2.0/) ![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/edgexfoundry/go-mod-secrets) [![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/edgexfoundry/go-mod-secrets)](https://github.com/edgexfoundry/go-mod-secrets/pulls) [![GitHub Contributors](https://img.shields.io/github/contributors/edgexfoundry/go-mod-secrets)](https://github.com/edgexfoundry/go-mod-secrets/contributors) [![GitHub Committers](https://img.shields.io/badge/team-committers-green)](https://github.com/orgs/edgexfoundry/teams/go-mod-secrets-committers/members) [![GitHub Commit Activity](https://img.shields.io/github/commit-activity/m/edgexfoundry/go-mod-secrets)](https://github.com/edgexfoundry/go-mod-secrets/commits)
- 
+
+## Delayed Start Go Build Tags
+
+The delayed start feature is conditional built with go build tags `delayedstart`.
+The default behavior is not included. If other go services or go module would like to build with this feature,
```

Comment:
So we will have to update every Device and App service to enable this be default. IMO, we should enable it by default and use this to disable it in edgex-go service that will never need it.

Reply:
not sure how you can disable the go-build tag in other build; the way i understand for go-build tags is for adding-in features ; i don't know how you can disable features from go build?

Reply:
@lenny-intel :  ok, thanks for the tips and advises and I reversed the go build tags and called `non_delayedstart`